model/channel_view_test.go

```diff
@@ -6,20 +6,17 @@ package model
 import (
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestChannelViewJson(t *testing.T) {
 	o := ChannelView{ChannelId: NewId(), PrevChannelId: NewId()}
 	json := o.ToJson()
 	ro := ChannelViewFromJson(strings.NewReader(json))
 
-	if o.ChannelId != ro.ChannelId {
-		t.Fatal("ChannelIdIds do not match")
-	}
-
-	if o.PrevChannelId != ro.PrevChannelId {
-		t.Fatal("PrevChannelIds do not match")
-	}
+	require.Equal(t, o.ChannelId, ro.ChannelId, "ChannelIdIds do not match")
```

Comment:
Would you please make all `require.Equal` to `assert.Equal`? The test can still continue, if something doesn't match

Reply:
sure